beets/ui/__init__.py

```diff
@@ -99,6 +99,12 @@ def print_(*strings):
     replaces it.
     """
     if strings:
+        if not isinstance(strings[0], basestring):
+            try:
+                strings = map(unicode, strings)
+            except UnicodeError:
+                strings = map(bytes, strings)
```

Comment:
Maybe that's not such a good idea, since it's confusingly different from the built-in `print` (although it does match the old `print_` behavior). So perhaps this is the best compromise.

I'm still worried about objects other than in the first position though. What about `print_(u'something', item)`? It seems like that will cause a ValueError when trying to join the strings.